src/core/config/ping_sender.h

```diff
@@ -48,13 +48,13 @@
 #endif
 
 /**
- * @def OPENTHREAD_CONFIG_PING_SENDER_DEFAULT_INTEVRAL
+ * @def OPENTHREAD_CONFIG_PING_SENDER_DEFAULT_INTERVAL
  *
  * Specifies the default ping interval (time between sending echo requests) in milliseconds.
  *
  */
-#ifndef OPENTHREAD_CONFIG_PING_SENDER_DEFAULT_INTEVRAL
-#define OPENTHREAD_CONFIG_PING_SENDER_DEFAULT_INTEVRAL 1000
+#ifndef OPENTHREAD_CONFIG_PING_SENDER_DEFAULT_INTERVAL
+#define OPENTHREAD_CONFIG_PING_SENDER_DEFAULT_INTERVAL 1000
```

Comment:
I know this is a typo, but to be safe, should we add a check for old name in `openthread-core-config-check`?